web/ui/react-app/src/pages/targets/JobsList.tsx

```diff
@@ -20,9 +21,17 @@ const JobsList: FC<JobsListProps & PathPrefixProps> = ({ filter, pathPrefix }) =
   const { response, error } = useFetch(`${pathPrefix}/api/v1/targets?state=active`);
 
   if (error) {
-    return <Error summary="Error fetching targets" message={error.message} />;
+    return (
+      <Alert color="danger">
+        <strong>Error fetching targets:</strong> {error.message}
+      </Alert>
+    );
   } else if (response && response.status !== 'success') {
```

Comment:
I think this shouldn't be handled by ```Joblist```

Reply:
Do you mean the line in particular, or all the error and loading handling? I can move it somewhere else, like Targets.tsx if you think that would be better :)

Reply:
...Or somewhere else. Let me know! Thanks!

Reply:
No no, I mean that components should not be aware of the response status. IMO they should be aware only of their data and nothing else. Actually this is what I've tried to solve here https://github.com/prometheus/prometheus/pull/6267 - creating strongly specific components responsible for doing one thing

Reply:
Okay, I understand what you mean. I can change it later to integrate with whatever Fetch logic exists in the future. For now, only `useFetch` exists.

Reply:
Yes, better to leave it for now :)

Reply:
cool thanks!